custom_components/homewhiz/appliance_controls.py

```diff
@@ -650,28 +743,34 @@ def extract_ac_control(controls: list[Control]) -> list[Control]:
     return controls
 
 
+# Only generate controls once to allow basic inter-Control communication
+controls: list[Control] = []
```

Comment:
@thecrisb This implementation with the global controls causes some pytests to fail. I could imagine this will cause issues when a user has multiple Homewhiz devices.